tests/test_predict.py

```diff
@@ -0,0 +1,14 @@
+import pytest
```

Comment:
You'll want to test predictions as well, by passing through a standard piece of text and making sure the output tensor looks like it's supposed to. Just some simple ones will do, and remember edge cases (like empty text, or text from multiple chapters, etc).

Reply:
I was able to write a test for the predict function by manually setting the seed in the test case.